Bus_Service/src/customer_action.py

```diff
@@ -20,14 +22,16 @@ def get_all_customer(self):
       print(e)
```

Comment:
the for loop can be written in a better way for better readability
    `for customer in customer:`